classroom/snippets/src/test/java/TestCancelGuardianInvitation.java

```diff
@@ -0,0 +1,31 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import com.google.api.services.classroom.model.GuardianInvitation;
+import org.junit.Assert;
+import org.junit.Test;
+
+// Unit test class for Cancel Guardian Invitation classroom snippet
+public class TestCancelGuardianInvitation {
+
+  @Test
+  public void testCancelGuardianInvitation() throws Exception {
```

Comment:
Do these tests actually run in some automated fashion or are they just for developer reference? Just thinking about if it makes sense to create an invitation in order to test cancelling it, sort of like create a topic to test deleting it https://github.com/mahima-desetty/java-samples/pull/1/files#diff-44f56e3fe96ed14e4d7d43af2193c04c6e3faec30ac7817ac29eb3fcecf93da5. Might be impractical as well so just checking

Reply:
Yeah I agree it makes more sense to create an invite and then cancel it. Updated!